### PR TITLE
Linear Search example in Java

### PR DESCRIPTION
**Pull request by khageshwor**
@thaneshwor
can you please review my code

### CODE REVIEW

File: Java/BinarySearch.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+    class BinarySearch{  
+        public static int BinarySearch(int arr[], int first, int last, int key){  
+
+            if (last >= first){  
+
+                int mid = first + (last - first)/2;
```
Comment (Thaneshwor) [inline Java/BinarySearch.java:6]: int mid = first + (last - first) / 2;

Comment (Thaneshwor) [inline Java/BinarySearch.java:6]: Add spaces after the operator

File: Java/BinarySearch.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+                return BinarySearch(arr, mid+1, last, key);     //search in right subarray  
+
+                }  
+            }  
+            return -1;  
+        }  
+        public static void main(String args[]){  
+
+            int arr[] = {10,20,30,40,50};
```
Comment (Thaneshwor) [inline Java/BinarySearch.java:28]: Add space after comma

Comment (Thaneshwor) [inline Java/BinarySearch.java:28]: Add space after comma

**Review by Thaneshwor — APPROVED** [review]

**Review by G1Joshi — APPROVED** [review]

**Comment by G1Joshi** [comment]
Welcome to the world of C0D1NG 🚀

**Pull request merged**